indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -655,7 +656,7 @@ def _process(self):
         return jsonify_data(html=_render_registration_details(self.registration))
 
 
-class RHRegistrationUploadFile(UploadRegistrationFileMixin, RHManageRegFormBase):
+class RHRegistrationUploadFile(UploadRegistrationFileMixin, RHManageRegistrationFieldActionBase):
```

Comment:
Should I make this base RH specific to the picture RH? If we make `self.field` available in the picture upload RH, it would make sense to do it for the file upload too. This would require changing the blueprints for file upload and `FileInput` component. Thoughts?

Reply:
yeah i think it makes sense to provide the same data for non-picture file upload as well